cpp/include/cudf/io/json.hpp

```diff
@@ -45,7 +45,7 @@ class json_reader_options_builder;
  * @brief Input arguments to the `read_json` interface.
  *
  * Available parameters and are closely patterned after PANDAS' `read_json` API.
```

Comment:
```suggestion
 * Available parameters are closely patterned after PANDAS' `read_json` API.
```
Can you help correct this typo also? Thanks!